src/pudl/output/eia860.py

```diff
@@ -1,13 +1,15 @@
 """Functions for pulling data primarily from the EIA's Form 860."""
 
+# import datetime
+
```

Comment:
Could you delete the commented out additions here and elsewhere in the file? They often end up being confusing later when folks see them and don't know if they're really supposed to be included or not. (and I see a #breakpoint() that I forgot to delete down there too, sorry!)